index.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio-website</title>
+  </head>
+  <body>
+    <header>
+      <img src=".png\geometry_1.png" />
+      <button class="menu">
+        <img src=".png\ic_menu.png" />
+      </button>
+    </header>
```

Comment:
To apply the correct semantics and improve your website navigation, please put your hamburger menu inside a `nav` element 😄.

You could have something like this:
```html
<header>
  <img src=".png\geometry_1.png" />

    <nav>
        <button class="menu">
            <img src=".png\ic_menu.png" />
        </button>
    </nav>
</header>
```